cgpsa.conf

```diff
@@ -574,23 +574,62 @@
 #
 # use_dma_spam_mailbox = false
 
-# The name of the mailbox to use for direct mailbox addressing when 
-# the "use_dma_spam_mailbox" setting is enabled. 
+# The name of the mailbox to use for direct mailbox addressing when
+# the "use_dma_spam_mailbox" setting is enabled.
 #
 # spam_mailbox_name = SPAM
 
 # A flag that determines whether CGPSA will discard spam messages above
 # a certain threshold (set in "discard_threshold"). This causes all
 # messages rated above the threshold level to be discarded without a
-# delivery attempt. 
+# delivery attempt.
 #
 # auto_discard = false
 
-# A threshold score above which CGPSA will discard messages if 
+# A threshold score above which CGPSA will discard messages if
 # "auto_discard" is enabled. For instance, with the default setting, any
 # message with a SpamAssassin score >= 25.0 will be discarded with no
 # delivery attempt if "auto_discard" is enabled.
 #
 # discard_threshold = 25.0
 
+# A flag that determines whether CGPSA adds an "X-Junk-Score" header
+# (see the "Junk Processing Simplified Rules" in the CGP manual). Th
+# header has a score generated based on the thresholds in the
+# "x_junk_score_thresholds" setting below. The default is false,
+# because it is possible that another filter that adds the "X-Junk-Score"
+# header may be used in conjunction wih CGPSA.
+#
+# x_junk_score_header = false
+
+# The thresholds for the "X-Junk-Score" header. These are six
+# space-separated numbers, each of which is a SpamAssassin score at
+# which the next "X" in an "X-Junk-Score: SCORE [XXX]" header is added,
+# where SCORE is a constant based on the number of Xs the email is given
+# and the "official" ranges for "X-Junk-Score" given at
+# https://www.communigate.ru/CGPKAS/ for the CGP Kaspersky plugin.
+#
+# For example, with the default settings, the following SpamAssassin
+# scores would result in the following headers:
+#
+# 0.0 - X-Junk-Score: 0 []
+# 1.0 - X-Junk-Score: 20 [X]
+# 4.0 - X-Junk-Score: 60 [XX]
+# 8.0 - X-Junk-Score: 85 [XXX]
+# 9.0 - X-Junk-Score: 93 [XXXX]
+# 9.5 - X-Junk-Score: 98 [XXXXX]
+# 10.0 - X-Junk-Score: 100 [XXXXXX]
```

Comment:
Maybe it can be clarified as below. Because it is not documented properly anywhere.

```
0.0 - X-Junk-Score: 0 []
1.0 - X-Junk-Score: 20 [X]
4.0 - X-Junk-Score: 60 [XX]    # Considered LOW
8.0 - X-Junk-Score: 85 [XXX]
9.0 - X-Junk-Score: 93 [XXXX]  # Considered MEDIUM
9.5 - X-Junk-Score: 98 [XXXXX] # Considered HIGH
10.0 - X-Junk-Score: 100 [XXXXXX]
```

Reply:
A bit verbose now, but okay.